Directory.Build.props

```diff
@@ -2,14 +2,22 @@
   <Import Project="eng\Versions.props" />
 
   <PropertyGroup>
+    <_MauiDotNetVersion Condition="'$(_MauiDotNetVersion)' == ''">6.0</_MauiDotNetVersion>
```

Comment:
We can set up our build pipelines to build for net6.0, net7.0, etc as variables, so we can dual produce/ship if need be.